api/policies/isInGame.js

```diff
@@ -16,5 +18,6 @@ module.exports = function (req, res, next) {
     }
     return res.badRequest({ message: `Error with user's session: 'game' must be an integer` });
   }
-  return res.badRequest({ message: `You can't do that because you're not a player in this game` });
+
+  return res.badRequest({ message: `You are not a player in this game!` });
```

Comment:
I had this as a forbidden earlier, which is what makes logical sense, but that kicks off the Reauthenticate logic clientside, since that is currently and incorrectly looking for a 403 to trigger reauth flow. Instead we should be using isLoggedIn to throw _401_ status codes when not authenticated and _that_ should be the trigger to reauthenticate.

401 - you're not authenticated; log in
403 - you _are_ authenticated but not allowed to do that e.g. spectate a game you're playing in or to make a move in someone elses' game